src/ray/core_worker/task_interface.cc

```diff
@@ -0,0 +1,26 @@
+#include "task_interface.h"
+
+namespace ray {
+
+Status CoreWorkerTaskInterface::CallTask(const RayFunction &function,
```

Comment:
would it be better to rename to `SubmitTask` and `SubmitActorTask`? 
or if you prefer to use call, maybe `CallRemote` and `CallRemoteActor` ?

Reply:
I didn't name it `SubmitTask` because there's also a `SubmitTask` in the `TaskTransport` interface. 
But that's probably not a big deal. I'll change them to `SubmitTask/SubmitActorTask`.